cni/deployments/kubernetes/Dockerfile.install-cni

```diff
@@ -8,13 +8,12 @@ jq='1.5*' \
 
 COPY istio-cni /opt/cni/bin/
 COPY istio-iptables /opt/cni/bin/
-COPY install-cni.sh /install-cni.sh
-COPY filter.jq /filter.jq
+COPY install-cni /usr/local/bin/
```

Comment:
Should the actual installation binary be left out of the `/opt/cni/bin` directory since it shouldn't need to be installed on the host's file-system? (adding this binary to that folder will install it to the host's file-system by default)

Reply:
Another +1 for keeping the new binary out of `/opt/cni/bin`. In certain distros (e.g. `cos_containerd`), if a directory like `/opt/cni/bin` is mounted by docker, it is mounted with `noexec`. If you put the install-cni binary there, it may prevent the installer from executing in those situations.